frontend/src/domain/Catalogue/Catalogue.tsx

```diff
@@ -57,15 +51,16 @@ export function Catalogue() {
   return (
     <Home breadcrumbs={breadcrumbs}>
       <Container>
-        <Header>{catalogue}</Header>
+        <Header>ACTION</Header>
         <Items>
-          {items.map(({ title, categories, price, cover }, index) => (
+          {catalogue.map(({ id, product_name, category, price}, index) => (
             <GameCard
               key={index}
-              title={title}
-              categories={categories}
+              id={id}
+              title={product_name}
+              category={category}
               price={price}
-              cover={cover}
+              cover={witcher}
```

Comment:
I this still a mock?

Reply:
Yes, covers will be added in a separate PR